caching/src/test/scala/com/velocidi/apso/caching/ExpiringLruCacheSpec.scala

```diff
@@ -85,8 +85,7 @@ class ExpiringLruCacheSpec(implicit ee: ExecutionEnv) extends Specification {
       cache(2)(Future.successful("B")) must beEqualTo("B").await
       cache(3)("C") must beEqualTo("C").await
       cache.store.toString === "{1=A, 2=B, 3=C}"
-      cache(4)("D")
-      Thread.sleep(10)
```

Comment:
I noticed that this test is now a bit more flaky.

I do think that we wanted the `await` here, not `Thread.sleep`.

Reply:
Yes. The `await` is more appropriate.